Explain the impact of unencrypted operations. Fixes #210 (#246)
mlswg/mls-architecture: draft-ietf-mls-architecture.md

perform add and remove operations. On the other hand, in many settings such as
open discussion forums, joining can be allowed for anyone.

> MLS handshake messages can be sent encrypted (in an MLS

Reviewer: ```suggestion
While MLS Application messages are always encrypted,
MLS handshake messages can be sent encrypted (in an MLS
```

perform add and remove operations. On the other hand, in many settings such as
open discussion forums, joining can be allowed for anyone.

While MLS Application messages are always encrypted,

> MLS handshake messages can be sent encrypted (in an MLS

Reviewer 2: ```suggestion
MLS handshake messages can be sent either encrypted (in an MLS
```

PrivateMessage) or unencrypted (in an MLS PublicMessage). Applications
may be designed such that intermediaries need to see handshake

> messages in order to enforce policy on which commits are allowed,
> and/or to provide MLS GroupInfo messages in a central location. If

Reviewer 2: ```suggestion
messages, for example to enforce policy on which commits are allowed,
or to provide MLS ratchet tree data in a central location. If
```

GroupInfo doesn't really have anything inherently private.  It's mainly the ratchet tree that is sensitive (in particular the credentials therein), and also the thing that is a pain to ship around.

handshake messages are unencrypted, it is especially important that
they be sent over a channel with strong transport encryption

> (see {{security-and-privacy-considerations}} in order to prevent external

Reviewer 3: ```suggestion
(see {{security-and-privacy-considerations}}) in order to prevent external
```

> attackers from monitoring the status of the group. Applications that

Reviewer 4: ```suggestion
attackers from monitoring the status of the group). Applications that
```
